public/js/components/App.js

```diff
@@ -58,8 +56,7 @@ const App = React.createClass({
 
   render: function() {
     return dom.div(
-      { className: classnames("debugger theme-body",
-                              { "theme-light": !isFirefoxPanel() }) },
+      { className: "debugger" },
```

Comment:
I pulled theme-body up to the Root element to be more consistent for other tools that use the toolbox, but I know we'll still want to figureout the toolbox / debugger api.